apps/storybook/src/forms/Checkbox.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Checkbox, Flex, Text } from "@optiaxiom/react";
+
+const meta: Meta<typeof Checkbox> = {
+  component: Checkbox,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Checkbox>;
+
+export const Basic: Story = {
+  args: {
+    children: "Label",
+  },
+};
+export const HelperText: Story = {
+  args: {
+    children: <>Label</>,
+    endDecorator: "Helper Text",
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    children: <>Label</>,
+    disabled: true,
+    endDecorator: "Helper Text",
+  },
+};
+export const Readonly: Story = {
+  args: {
+    children: <>Label</>,
+    endDecorator: <>Helper Text</>,
+    readonly: true,
+  },
+};
```

Comment:
let's not add the `endDecorator` to other stories - and only have a single dedicated story showcasing end decorators